spec/latest/core/core.jsonld

```diff
@@ -431,6 +432,16 @@
       "@id": "hydra:template",
       "@type": "rdf:Property",
       "label": "template",
+      "comment": "A templated string with placeholders. The literal's datatype indicates the template syntax; if not specified, hydra:rfc6570Template is assumed.",
+      "seeAlso": "hydra:rfc6570Template",
+      "domain": "hydra:IriTemplate",
+      "range": "rdfs:Literal",

```

Comment:
Agree on dropping the domain.
I strongly recommend to keep the range. All this says is that it's a literal property, and I know only one case where it makes sense to allow both objects and literals as a property's value (`rdf:object`).

Amended as 6df5dfc49b677ccc21d68e2a1c5c725954a532a1.